spec/acceptance/docker_basic_crud_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'spec_helper_acceptance'
+
+describe 'docker CRUD tests' do
+  context 'docker add' do
+    it 'installs docker, pulls images and runs container' do
```

Comment:
This should probably be a before block for this context. It's really about setting up the assertions I think.